.github/ISSUE_TEMPLATE/post_vote_checklist.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Post Vote Checklist
+about: A checklist for tracking post-vote release tasks
+title: 'Post-vote release checklist for version [e.g. 1.0.0]'
+labels:
+assignees: ''
+
+---
+
+- [ ] Label this issue with the GitHub [milestone](https://github.com/apache/accumulo-access/milestones) that corresponds to this release version
+- [Git](https://github.com/apache/accumulo-access) tasks
+  - [ ] Create a signed `rel/<version>` tag (and push it)
+  - [ ] Merge `<version>-rc<N>-next` branch into a maintenance branch (if maintenance is expected),
+        and then into the `main` branch (and push them)
+  - [ ] Remove `*-rc*` branches
+  - [ ] Update the Accumulo-Access dependency versions in the contrib projects
+- [Nexus](https://repository.apache.org) tasks
+  - [ ] Release the staging repository corresponding to the successful release candidate (use "release" button)
+  - [ ] Drop any other staging repositories for Accumulo-Access (do *not* "release" or "promote" them)
+- [SVN / dist-release](https://dist.apache.org/repos/dist/release/accumulo-access) tasks
+  - [ ] Upload the release artifacts (tarballs, signatures, and `.sha512` files) for the mirrors
+  - [ ] Remove old artifacts from the mirrors (**after** updating the website)
+- [Board reporting tool](https://reporter.apache.org/addrelease?accumulo)
+  - [ ] Add the date of release (the date the release artifacts were uploaded to SVN `UTC+0000`)
+- Verify published artifacts
+  - [ ] In [Maven Central](https://repo1.maven.org/maven2/org/apache/accumulo/accumulo-access/)
+  - [ ] In [ASF Downloads](https://downloads.apache.org/accumulo)
+  - [ ] In [several mirrors or CDN](https://www.apache.org/dyn/closer.lua/accumulo)
+- Update the [website](https://accumulo.apache.org/)
+  - [ ] Release notes
+  - [ ] Add `LTM: true` to the release notes front-matter for LTM releases
+  - [ ] Navigation
+  - [ ] Downloads page
+  - [ ] If LTM release, update previous LTM release entry on downloads page and release notes with an EOL date 1 year from the current release date
+  - [ ] DOAP file
```

Comment:
We'll create a new DOAP file just for this library, separate from the main accumulo one. We have precedent for this, with the maven plugin. See the doap directory for details. We currently don't have docs or navigation about this library, or anything on the download page. We'll probably want to add that at some point after the first release.